chainer/links/connection/convolution_2d.py

```diff
@@ -179,8 +191,11 @@ def forward(self, x):
         """
         if self.W.array is None:
             self._initialize_params(x.shape[1])
+        W = self.W
+        if self.standardizeW:
+            W = weight_standardization.weight_standardization(W, self.eps)
```

Comment:
I'd like to implement the feature by `LinkHook`.  See #5742 for example.  It's OK to separate a PR for it (in order to add `F.weight_standardization` first).

Reply:
https://github.com/crcrpar/chainer/blob/weight-standardization-hook/chainer/link_hooks/weight_standardization.py
Though it's in progress, I have tried to implement this in Link hook, so feel free to copy this if helpful.

Reply:
Thanks @crcrpar !
Your code is very helpful for my new implementation in `link_hook`.